src/scout_map.py

```diff
@@ -1438,7 +1443,7 @@ def buffer_distance(self, point_details, all_points):
             if not valid:
                 distance = 0
         else:
-            distance = 0
+            distance = all_points["buffer_distance"]
```

Comment:
I'm not sure this is correct, I thought that `all_points["buffer_distance"]` was a column of a dataframe, whereas distance must be a number?

Reply:
Resolved. Merged master back into branch to replace with correct expression